src/tests/nativeaot/SmokeTests/StackTraceMetadata/StackTraceMetadata.csproj

```diff
@@ -12,4 +12,8 @@
     <Compile Include="StackTraceMetadata.cs" />
     <Compile Include="BodyFoldingTest.cs" />
   </ItemGroup>
+
+  <ItemGroup>
+    <IlcArg Include="--export-unmanaged-entrypoints" />
```

Comment:
Set `IlcExportUnmanagedEntrypoints` property to true instead of using the command line option directly?

It is what the other tests do: https://github.com/dotnet/runtime/blob/3f1cfbba1f3c6a6b2e04813dc6b905d04fcc172d/src/tests/nativeaot/GenerateUnmanagedEntryPoints/GenerateUnmanagedEntryPoints.csproj#L5

Reply:
The reason I did it this way is because we're not really testing the thing is exported, just making sure linker doesn't error out.

If I use the property and we rename or delete the property, the test silently stops testing this. If it stays command line argument, the test build will fail if we ever rename or delete it.

But I can change it if you think it's not a risk worth mitigating.